build.yaml

```diff
@@ -10,12 +10,24 @@ cassandra:
 build:
   - type: maven
     version: 3.2.5
-    goals: clean compile test --fail-never -Plong
+    goals: clean compile test integration-test --fail-never -Plong
```

Comment:
Cool, so that'd run both test and integration-test phase (as well as any other checks?).  I'll give that a try, thanks!